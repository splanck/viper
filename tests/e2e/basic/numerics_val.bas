10 PRINT VAL("  -12.5E+1x")
20 PRINT VAL("abc")
30 PRINT VAL("  +12.34E1x")
