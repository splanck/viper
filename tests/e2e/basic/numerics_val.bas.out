-125
0
123.4
